lib/utils/autocomplete.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const homedir = require('os').homedir();
+const readFileSync = require('./fs/readFileSync');
+const fileExistsSync = require('./fs/fileExistsSync');
+
+const tab = require('tabtab')({
+  name: 'serverless',
+});
+
+const autocomplete = () => {
+  const cacheFilePath = `${homedir}/.serverless/autocomplete.json`;
+  if (!fileExistsSync(cacheFilePath)) {
+    return;
```

Comment:
@eahefnawy this doesn't seem right. Instead of bailing out of autocomplete if the cache file doesn't exist, instead it should run `serverless.init()` which should generate the cache file for you.